### PR TITLE
Module 2: Hello Microverse Project

### PR DESCRIPTION
**Pull request by LIBERCOSOFT**
##Hello Micoverse Project
In this Module, I created a:
- HTML file that includes a header with the text "Hello Microverse!".
- CSS file that includes one class with styles for your header (make it in your favorite color).
- A test.md file with the text "This file should be ignored by git".

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/test.md
+/.github
```
Comment (ShahierNashaat) [inline .gitignore]: - [x] Kindly be noticed that the .gitignore file listing `test.md` file and `node_modules` directory.

File: README.md [hunk 2]
```diff
@@ -1 +1,45 @@
...
+
+### Setup
+Create the directory in which you'll like to clone this project.
+
+### Install
+Navigate to the created directory above
+Run git clone 'the project's https or ssh'
+
+### Usage
+View the HTML document in your preferred browser
```
Comment (ShahierNashaat) [inline README.md]: - [x] Kindly in your README file write the steps that the user should follow to clone your repo ie.
- open your terminal and write `git clone https://github.com/LIBERCOSOFT/hello-microverse.git`.
- go to the `hello-microverse` directory.
- etc.

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @LIBERCOSOFT, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 1 above)
Comment (LIBERCOSOFT) [reply .gitignore]: Thanks for the feedback, so I should add only `test.md` and `node_modules` to my .gitignore file?

Comment (ShahierNashaat) [reply .gitignore]: That's right.

Comment (LIBERCOSOFT) [reply .gitignore]: Thanks

**Review by Ismailco — APPROVED** [review]
### Hi @LIBERCOSOFT, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**